google-pubsub.yml

```diff
@@ -83,10 +83,10 @@ bind:
       default: ${config("gcp.project")}
   computed_inputs:
     - name: service_account_name
-      default: ${str.truncate(20, "pcf-binding-${request.binding_id}")}
+      default: ${str.truncate(20, "csb-binding-${request.binding_id}")}
       overwrite: true
     - name: service_account_display_name
-      default: ""
+      default: ${str.truncate(20, "csb-binding-${request.binding_id}")}
```

Comment:
question: is there any length limitation? Why do we use 20 characters?

Reply:
The default was empty which resulted in the service account having no name in the console. Would be nice to do the same for other services because this is a calculated value so the customer cannot pass it in.
The truncation tbh was copied from other services -so they all use the 20 char limit. The length limit is between 6 and 30 characters.

Reply:
Thank you very much. I am happy with the change because it improves the UX.